bindings/python/examples/wallet/offline_signing/0_generate_addresses.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+# In this example we generate addresses which will be used later to find
+# inputs.
+
+from iota_sdk import Wallet, StrongholdSecretManager, CoinType, ClientOptions
+from dotenv import load_dotenv
+import json
+import os
+
+load_dotenv()
+
+OFFLINE_WALLET_DB_PATH = "./wallet/offline_signing/example-offline-walletdb"
+STRONGHOLD_SNAPSHOT_PATH = "./wallet/offline_signing/example.stronghold"
+ADDRESSES_FILE_PATH = "./wallet/offline_signing/example.addresses.json"
+
+
+node_url = os.environ.get('NODE_URL', 'https://api.testnet.shimmer.network')
+offline_client_options = ClientOptions()
+
+if 'STRONGHOLD_PASSWORD' not in os.environ:
+    raise Exception(".env STRONGHOLD_PASSWORD is undefined, see .env.example")
+
+secret_manager = StrongholdSecretManager(
+    STRONGHOLD_SNAPSHOT_PATH, os.environ['STRONGHOLD_PASSWORD'])
+
+wallet = Wallet(OFFLINE_WALLET_DB_PATH, offline_client_options,
+                CoinType.IOTA, secret_manager)
+
+if 'MNEMONIC' not in os.environ:
+    raise Exception(".env MNEMONIC is undefined, see .env.example")
+
+# Store the mnemonic in the Stronghold snapshot, this only needs to be
+# done once
+wallet.store_mnemonic(os.environ['MNEMONIC'])
+
+account = wallet.create_account('Alice', "rms")
+print("Account created:", account.get_metadata())
+
+addresses = account.addresses()
```

Comment:
If this generates addresses, shouldn't it be named appropriately?

Reply:
No, it's just returning the account addresses

Reply:
The example says otherwise. Change that then?

Reply:
The address is generated when you create the account, do you want me to rename to `generate_address`?

Reply:
Hmm. Maybe just be more explicit in the comment.

Reply:
Added a comment, better now?

Reply:
Not really 😆 this is the comment I am talking about, above.

```rust
# In this example we generate addresses which will be used later to find
# inputs.```

Reply:
Ah already wondered why you wrote `comment` :sweat_smile: 
Updated it